utils/warnings.ml

```diff
@@ -441,13 +441,11 @@ let message = function
   | Unused_match -> "this match case is unused."
   | Unused_pat   -> "this sub-pattern is unused."
   | Instance_variable_override [lab] ->
-      "the instance variable " ^ lab ^ " is overridden.\n" ^
-      "The behaviour changed in ocaml 3.10 (previous behaviour was hiding.)"
+      "the instance variable " ^ lab ^ " is overridden."
```

Comment:
The fact 3.10 was a long time ago doesn't mean there is no old code left around.
(Yet there was no warning for the previous transition, from OCaml 1 to OCaml 2.00, when the behavior changed from overriding to shadowing.)